guides/user-manual/api_auth.rst

```diff
@@ -17,33 +17,53 @@ There a few options for authenticating against the M&M API:
 
 .. _api-login-command:
 
-Login command
+Login Command
 -------------
 
-This method is available in all versions of M&M, and all the different flavors of the M&M API (SOAP/JSON-RPC/REST). In the REST API, this command is available via the URL commands/Login
+This method is compatible with all versions of Micetro, and all every variant of the Micetro API (SOAP/JSON-RPC/REST). In the REST API, this command is accessible via the URL commands/Login
 
-The Login command takes in server (for the M&M Central server), user name, and password, and returns a session ID, which should then be used for all other API calls via their session parameter.
+The Login command requires a server (for the Micetro Central server), username, and password, and it returns a session ID. This session ID should be used for all other API calls via the session parameter.
 
-The Login command and its parameters is fully documented in the normal SOAP API documentation
+The Login command and its parameters are fully documented in the standard SOAP API documentation
 
 .. note::
-  Remember to configure HTTPS for the mmws and/or _mmwebext sites for this authentication method, since the username and password can easily be extracted for anyone listening in on the communication.
+  Remember to configure HTTPS for the mmws and/or _mmwebext sites when using this authentication method to prevent usernames and passwords from being intercepted during communication.
 
 .. _api-login-headers:
 
 Authorization Headers
 ---------------------
 
-With the introduction of Men&Mice Web Services and the REST API in version 7.3, it's now also possible to authenticate via HTTP headers, by adding a Basic Authorization header, or with a Negotiate Authorization header, if the client supports it.
+It's possible to authenticate via HTTP headers by adding a Basic Authorization header or a Negotiate Authorization header, if supported bye the client.
 
 .. note::
-  If AuthorizationMethods is not defined in the preference file mmWS will only accept basic authorization.
+  If AuthorizationMethods is not defined in the preference file, mmWS will only accept Basic Authorization.
 
-By using authorization headers for authentication, the Login command becomes unnecessary, and the session ID is not used. The "session" parameter for all the commands must be omitted, and not left blank.
+Using authorization headers for authentication eliminates the need for the Login command, which makes the session ID unnecessary. The "session" parameter should be omitted entirely from all commands, not just left blank.
 
-With this authentication method, multiple M&M Web Service instances can be deployed and used simultaneously behind a load balancer to provide a redundancy that is transparent to the API client.
+This method of authentication allows multiple Micetro Web Interfaces to be deployed behind a load balancer, providing redundancy that is seamless to the API client.
```

Comment:
remove this - this is now the case for all auth mechanisms

Reply:
and the next section

Reply:
yes, remove all of it. All our authentication mechanisms now allow multiple Micetro Web Interfaces and mmWS endpoints to be deployed behind a load balancer.

Maybe leave this statement though:  "The authorization header authentication method is only available for JSON-RPC and REST APIs". I assume it is accurate.